app/buyers/views/buyers.py

```diff
@@ -307,6 +308,9 @@ def prepared_response_contents_for_brief(brief, responses):
     ess_req_names = brief.get(ESS, [])
     nth_req_names = brief.get(NTH, [])
 
+    def csv_cell_sanitize(text):
+        return re.sub(r"^(=|\+|-|@|!|\|{|}|\[|\]|<)+", '', unicode(text).strip())
+
```

Comment:
To handle bool type objects which implicitly were converted to string by the csv writer